### PR TITLE
DROOLS-1799: [GDT] Enable field binding for formula calculation type

### PR DESCRIPTION
**Pull request by jomarko**
Ensemble with:
- https://github.com/kiegroup/drools/pull/1608

@manstis @karreiro I will be pleased with your review.

### CODE REVIEW

**Comment by jomarko** [comment]
jenkins please retest this

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/OperatorPageTest.java [hunk 1]
```diff
@@ -318,6 +318,19 @@ public void testFilterOptionsForConstraintTypeLiteralWhenConstraintValueIsNotLit
...
     }
 
+    @Test
+    public void testFilterOptionsForConstraintTypeLiteralWhenConstraintValueIsFormula() {
+        when(plugin.constraintValue()).thenReturn(BaseSingleFieldConstraint.TYPE_RET_VALUE);
+
+        final String[] result = page.filterOptionsForConstraintTypeLiteral(OperatorsOracle.EXPLICIT_LIST_OPERATORS);
+        final List<String> operators = Arrays.asList(result);
+
+        assertFalse(operators.contains("in"));
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/OperatorPageTest.java]: Are these "contains" assertions necessary? You're checking ```operators.size() == 0```

**Comment by jomarko** [comment]
@manstis I added commit with updated tests

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please execute full downstream build.

**Comment by karreiro** [comment]
Jenkins please execute full downstream build.

**Comment by karreiro** [comment]
Jenkins please execute full downstream build.

**Pull request merged**